x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -243,6 +244,23 @@ private static String[] enterpriseSearchAcknowledgementMessages(OperationMode cu
         return Strings.EMPTY_ARRAY;
     }
 
+    private static String[] esqlAcknowledgementMessages(OperationMode currentMode, OperationMode newMode) {
+        switch (newMode) {
+            case BASIC:
+            case STANDARD:
+            case GOLD:
+                switch (currentMode) {
+                    case PLATINUM:
+                        return new String[] {};
+                    case TRIAL:
+                    case ENTERPRISE:
+                        return new String[] { "ES|QL cross-cluster search requires an enterprise license." };
```

Comment:
I see that other checks use the future tense ("will be disabled") while this uses the present tense ("requires"). Should we sync it?

Reply:
Changed to "ES|QL cross-cluster search will be disabled".